### ISSUE TITLE
youtube.com: ads

### ISSUE THREAD

**Issue by Jebesprijave** [uBlockOrigin/uAssets#10697]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/143316083-5dd9bf87-ee9a-4ac2-a5a0-131c56c65551.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png)
- [X] I have no custom filter lists, or I verified that the issue still occurs without custom filter lists
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625)
- [X] I did not open more than five (5) issues today, or I am a long time contributor to this repo

### URL address of the web page

`https://www.youtube.com/`

### Category

ads

### Describe the issue

When I scroll don on the Home page, the position of the video thumbnails shifts because an empty space appears in the upper right corner of the video list. If I disable uBlock on Youtube, the empty space is filled with an ad.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.39.0
Firefox: 94
filterset (summary): 
  network: 78070
  cosmetic: 39320
  scriptlet: 16109
  html: 598
listset (total-discarded, last updated): 
  default: 
    ublock-filters: 30263-27, 2d.4h.48m
    ublock-badware: 3494-1, 2d.4h.45m
    ublock-privacy: 184-0, 2d.4h.43m
    ublock-abuse: 74-0, 2d.4h.41m
    ublock-unbreak: 1707-0, 2d.4h.39m
    easylist: 61282-545, 2d.4h.37m
    easyprivacy: 26396-474, 2d.4h.35m
    urlhaus-1: 8190-0, 50m
    plowe-0: 3697-3, 2d.4h.31m
filterset (user): [array of 7 redacted]
modifiedUserSettings: 
  showIconBadge: false
modifiedHiddenSettings: [none]
supportStats: 
  launchToReadiness: 181642
  launchFromSelfie: true
```
</details>

**Comment by uBlock-user** [comment]
Duplicate of https://github.com/uBlockOrigin/uAssets/issues/7636

**Comment by gorhill** [comment]
> To avoid burdening VOLUNTEERS with invalid or duplicate reports, disrespecting these steps WILL result in your account being blocked on this repo (temporarily, then permanently with repeated disregard for the template).

> - [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/143316083-5dd9bf87-ee9a-4ac2-a5a0-131c56c65551.png) to find out)

<https://github.com/uBlockOrigin/uAssets/issues?q=is%3Aissue+sort%3Aupdated-desc+%22youtube.com%22+in%3Atitle>
